src/reactable/table.jsx

```diff
@@ -133,6 +133,36 @@ export class Table extends React.Component {
         this.tfoot = tfoot;
 
         this.initializeSorts(props);
+        this.initializeFilters(props);
+    }
+
+    initializeFilters() {
+        this._filterable = {};
+        // Transform filterable properties into a more friendly list
+        for (let i in this.props.filterable) {
+            let column = this.props.filterable[i];
+            let columnName, filterFunction;
+
+            if (column instanceof Object) {
+                if (typeof(column.column) !== 'undefined') {
+                    columnName = column.column;
+                } else {
+                    console.warn('Filterable column specified without column name');
+                    return;
```

Comment:
you're right. i was basing this really closely on initializeSort (since i want the behavior to be as similar as possible). i'll change this though